neuromation/api/client.py

```diff
@@ -50,6 +51,9 @@ def __init__(
         self._images: Optional[Images] = None
 
     async def close(self) -> None:
+        if self._closing:
+            return
+        self._closing = True
         await self._core.close()
```

Comment:
If any of operation will fail, would not client keep some unclosed resources? Is it worth to guarantee closing all resources?
```python
    try:
        close_resource1()
    finally:
        try:
            close_resource2()
        finally:
            close_resource3()
```
or
```python
    try:
        try:
            close_resource1()
        finally:
            close_resource2()
    finally:
        close_resource3()
```

Reply:
Good question.
aiohttp closes client session and open socket objects on GC but does it in *dirty style*: with warnings etc.

I think it is enough, at least until people start objecting.